src/Elemental/Form/Field/LongText.elm

```diff
@@ -104,5 +109,7 @@ view options model =
         , placeholder = options.placeholder
         , height = options.height
         , onInput = ChangedInput
+        , maybeInteractionConfig =
```

Comment:
The naming isn't consistent. Here you used `maybeInteractionConfig` and in the others you used `maybeOnInteraction`. Also, I prefer the name `maybeInteractionConfig`.